views/funcoes_request_response.py

```diff
@@ -223,8 +223,12 @@ def listar_turmas_aluno(request, id_aluno):
     turmas = gerenciador_turmas_alunos.listar_turmas_aluno(id_aluno)
     return JsonResponse(turmas)
 
-
-def listar_detalhes_ciclos_por_id_turma(request, id_turma):
-    turma = gerenciador_turmas.obter_turma(id_turma)
-    resposta = gerenciador_ciclos.detalhesCicloTurma(turma, id_turma)
+def listar_detalhes_ciclos_por_id_turma(request):
+    resposta = {}
+    turmas = gerenciador_turmas.busca_turmas()
+    for id_turma, turma_info in turmas.items():
+        #Devolve um objeto onde a chave é o id da turma
```

Comment:
Se quiser documentar essas questões, vc deve usar esse padrão de docstring. https://dev.to/suspir0n/o-segredo-de-uma-documentacao-simples-em-python-4ik9